src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/incentives/IncentivesServiceTest.kt

```diff
@@ -25,14 +26,14 @@ internal class IncentivesServiceTest {
   private val updateQueueService: IncentivesUpdateQueueService = mock()
   private val telemetryClient: TelemetryClient = mock()
   private val incentivesService =
-    IncentivesService(incentiveApiService, nomisApiService, mappingService, updateQueueService, telemetryClient)
+    IncentivesService(incentiveApiService, nomisApiService, mappingService, updateQueueService, telemetryClient, objectMapper())
 
   @Nested
   inner class CreateIncentive {
 
     @Test
-    fun `should log a processed visit booked event`() {
-      whenever(incentiveApiService.getIncentive(123)).thenReturn(newIncentive())
+    fun `should log a processed visit booked event`() = runBlocking {
+      whenever(incentiveApiService.getIncentive(123)).thenReturn(newIncentive(id = 123))
```

Comment:
tests had conflicting data between what was in event and what was in mock, so corrected since we now use the event id for telemetry (so we can bail out early if there is a duplicate create)